shared/common-adapters/badge.native.js

```diff
@@ -48,4 +48,49 @@ const styles = styleSheetCreate({
   },
 })
 
-export default Badge
+export class Badge2 extends React.Component<Badge2Props> {
+  static defaultProps = {fontSize: 11, radius: 12, leftRightPadding: 6, topBottomPadding: 4}
+
+  render() {
+    return (
+      <Box
+        style={collapseStyles([
+          badge2Styles.badge,
+          {
+            borderRadius: this.props.radius,
+            height: this.props.radius * 2,
```

Comment:
i think a prop of height is clearer than a radius. a designer will know what height means but radius is more confusing

Reply:
i don't think we need leftRightPadding or topBottomPadding to be props. can't we drive this with just the fontSize and height?

Reply:
i hope 99% of the usage of this will be something like
`<Badge2 fontSize={14} />`
where radius is calculated automatically and can be overridden if people want more control

Reply:
If a user is just defining `fontSize`, does that mean the radius would be calculated as `radius = lineHeight + topPadding + bottomPadding`?

Reply:
i mean, i should be able to just set fontsize and get an appropriate padding w/o overriding it. whatever that is, we'd have to play with it. maybe it makes sense for it to be fixed, maybe it scales up based on the fontsize, i'd have to see it